### PR TITLE
[zh_CN] follow en changes

### PR DESCRIPTION
**Pull request by blackpuppy**

### CODE REVIEW

File: zh/development/configuration.rst [hunk 1]
```diff
@@ -290,12 +291,12 @@ Session
     * 'database' - 使用 CakePHP 的数据库会话。
     * 'cache' - 使用 Cache 类保存会话。
 
-    要定义自定义会话处理器，把它保存在 
-    ``app/ModelSDatasource/Session/<name>.php`` 中。确保这个类实现了 
-    :php:interface:`CakeSessionHandlerInterface`，并设置 Session.handler 为 
+    要定义自定义会话处理器，把它保存在
+    ``app/ModelSDatasource/Session/<name>.php`` 中。确保这个类实现了
```
Comment (markstory) [inline zh/development/configuration.rst]: I think `ModelSDatasource` is supposed to be `Models/Datasource`

**Comment by markstory** [comment]
Looks great other than that 1 minor issue.

File: zh/development/configuration.rst (hunk 1 above)
Comment (blackpuppy) [inline zh/development/configuration.rst]: Ah!  An error lurked for over a year.  Thanks for picking it up! 👍

**Comment by markstory** [comment]
Great work! Thanks again :smile:

**Pull request merged**